developer/js/package.json

```diff
@@ -15,7 +15,9 @@
   },
   "homepage": "https://github.com/keymanapp/keyman#readme",
   "bin": {
-    "kmlmc": "dist/cli.js"
+    "kmlmc": "dist/cli.js",
```

Comment:
We may want to rename `cli.js` to `kmlmc.js`, to match the rest.

This is one of the things I'll look at months later and be like... why is this one named different than the rest?

Reply:
👍